src/Http/Cookie/Cookie.php

```diff
@@ -160,6 +176,17 @@ public function __construct(
         $this->expiresAt = $expiresAt;
     }
 
+    /**
+     * Set default options for the cookies.
+     *
+     * @param array $options Default options.
+     * @return void
+     */
+    public static function setDefaults(array $options): void
+    {
+        static::$defaults = $options + static::$defaults;
```

Comment:
If you set defaults twice with a sparse array, do you expect it to reset the other fields or merge with previously set defaults?